styles/helpers/typography.css

```diff
@@ -96,26 +96,22 @@ body {
   }
 }
 
-
 .heading--light {
   color: var(--color-heading-light);
 }
+
 .text-color-body {
   color: var(--color-body);
 }
+
 .text-color-heading {
   color: var(--color-heading);
 }
+
 .text-color-light {
   color: var(--color-heading-light);
```

Comment:
this is a little misleading of a helper

Reply:
What do you mean?

Reply:
the class name is text color light, but the variable is color heading light